### DISCUSSION TITLE
Tutorial Station

### DISCUSSION THREAD

**Discussion by ZachZent** [tsunamayo/Starship-EVO#3957] [Q&A]
Moving the conversation from https://github.com/tsunamayo/Starship-EVO/issues/3906 to here.

So we know the tutorial station will be modular, large, and the actual tutorial part will be an empty module for the actual tutorial bit to be copied in. The question before we start building is what general style of station should it be.

Just to be clear, this isn't saying that whatever station is made should be a direct copy of any example, just what general (exterior) style the station should be going for. A realistic station is a far cry from a geometric station.

More utilitarian sci-fi design like halo and this unity design?
![image](https://user-images.githubusercontent.com/36980446/124800053-0f0b5500-df0a-11eb-8c12-ee84cf19e5f0.png)
![image](https://user-images.githubusercontent.com/36980446/124800148-2a766000-df0a-11eb-9da5-8842ef183dad.png)

Bit smoother like the Star Wars station or this random one?
![image](https://user-images.githubusercontent.com/36980446/124800292-5560b400-df0a-11eb-93c9-e148b338f985.png)
![image](https://user-images.githubusercontent.com/36980446/124800305-598cd180-df0a-11eb-83d2-675fd145c449.png)

Realistic with solar panels and rings?
![image](https://user-images.githubusercontent.com/36980446/124800683-ca33ee00-df0a-11eb-9a33-11c39f8fc511.png)
![image](https://user-images.githubusercontent.com/36980446/124800915-10894d00-df0b-11eb-8b34-04f812af2575.png)

Ill call this style "Utopian"?
![image](https://user-images.githubusercontent.com/36980446/124801065-3f9fbe80-df0b-11eb-9f83-651fcacbbe34.png)
![image](https://user-images.githubusercontent.com/36980446/124801080-462e3600-df0b-11eb-8456-dfdc8cbba166.png)

Bulky?
![image](https://user-images.githubusercontent.com/36980446/124801186-6bbb3f80-df0b-11eb-8e91-54f74e33abde.png)
![image](https://user-images.githubusercontent.com/36980446/124801233-783f9800-df0b-11eb-93f7-494aac963631.png)

Ring stations because there are so many of them that they may be its own style?
![image](https://user-images.githubusercontent.com/36980446/124801461-b9d04300-df0b-11eb-8bc6-2e070bba9eda.png)
![image](https://user-images.githubusercontent.com/36980446/124801525-c9e82280-df0b-11eb-8c19-dee3274a3ae0.png)

Geometric like Elite Dangerous or No Mans Sky?
![image](https://user-images.githubusercontent.com/36980446/124801561-d4a2b780-df0b-11eb-866f-b66ec6093d80.png)
![image](https://user-images.githubusercontent.com/36980446/124801651-ec7a3b80-df0b-11eb-9c46-76ea054812cc.png)

There are more obviously, but this should be a good spattering of styles

**Comment by tsunamayo** [comment]
@ZachZent Build whatever style you feel comfortable in. For the tuto I will simple re-use a station hub. I want it attached to the rest of a normal station, so that the player can right away buy a ship and tackle a few missions.
So if you want to help in that regard just build me some stations modules!
